### PR TITLE
Console: gentpch command

### LINKED ISSUE #1137: show tables is broken
```
Hyrise is running a (debug) build.

(debug)> show tables;
libc++abi.dylib: terminating with uncaught exception of type std::logic_error: src/lib/logical_query_plan/abstract_lqp_node.cpp:180 Can only forward input expressions, if there is only a left input
```

Also, apparently untested.

### PR DESCRIPTION
**Pull request by mrzzzrm**
`generate` became `gentpcc`; `gentpch` was newly introduced. Removed some confusingly obvious out()put like "After TPC-C tables are generated, SQL queries can be executed".  

Fixes #1137

### CODE REVIEW

File: src/bin/console/console.cpp [hunk 1]
```diff
@@ -760,8 +796,8 @@ char** Console::_command_completion(const char* text, int start, int end) {
   // Choose completion function depending on the input. If it starts with "generate",
   // suggest TPC-C tablenames for completion.
   const std::string& first_word = tokens.at(0);
-  if (first_word == "generate") {
-    // Completion only for two words, "generate", and the TABLENAME
+  if (first_word == "gentpch") {
+    // Completion only for two words, "gentpch", and the TABLENAME
```
Comment (mrks) [inline src/bin/console/console.cpp]: Did anyone ever use the "generate single table" feature? If not, drop it?

File: src/bin/console/console.cpp [hunk 2]
```diff
@@ -760,8 +796,8 @@ char** Console::_command_completion(const char* text, int start, int end) {
   // Choose completion function depending on the input. If it starts with "generate",
   // suggest TPC-C tablenames for completion.
   const std::string& first_word = tokens.at(0);
-  if (first_word == "generate") {
-    // Completion only for two words, "generate", and the TABLENAME
+  if (first_word == "gentpch") {
```
Comment (mrks) [inline src/bin/console/console.cpp]: Personally, I would like `generate tpc[ch]` better. Newcomers to the database benchmarking world will have no idea what `gentpch` might mean (or how to pronounce it :D )

File: src/lib/logical_query_plan/show_tables_node.cpp [hunk 3]
```diff
@@ -12,6 +16,15 @@ std::shared_ptr<AbstractLQPNode> ShowTablesNode::_on_shallow_copy(LQPNodeMapping
   return ShowTablesNode::make();
 }
 
+const std::vector<std::shared_ptr<AbstractExpression>>& ShowTablesNode::column_expressions() const {
```
Comment (mrks) [inline src/lib/logical_query_plan/show_tables_node.cpp]: ?

Comment (mrzzzrm) [reply src/lib/logical_query_plan/show_tables_node.cpp]: Ran into #1137 as well, fixed it and didn't mention it ;)

File: src/bin/console/console.cpp (hunk 2 above)
Comment (mrzzzrm) [reply src/bin/console/console.cpp]: I turned it into `generate_tp{ch}`, is that okay? `generate tpch` with a fixed singlespace between "generate" and "tpch" would work as well, but that makes it look like we have some kind of sub-command system and we don't have that

Comment (mrks) [reply src/bin/console/console.cpp]: ok

**Pull request merged**